harvester/edurep/extraction.py

```diff
@@ -219,3 +219,9 @@ def get_disciplines(cls, soup, el):
     def get_humanized_disciplines(cls, soup, el):
         blocks = cls.find_all_classification_blocks(el, "discipline", "czp:entry")
         return list(set([block.find('czp:langstring').text.strip() for block in blocks]))
+
+    @classmethod
+    def get_analysis_allowed(cls, soup, el):
+        node = el.find('czp:copyrightandotherrestrictions')
+        value = node.find('czp:value').find('czp:langstring').text.strip() if node else False
+        return value and "-nd" not in value
```

Comment:
moeten we hier ook niet checken voor `"no"`?

Reply:
Volgens mij niet. "no" staat niet in het lijstje met possible values volgens pivotal, alhoewel als ik kijk naar het stukje code waar jouw volgende comment over gaat... In pivotal staat ook: We do not want to analyze records if they have a "non-derivative clause": in that case the string will contain "-nd". Ik zou juist zeggen dat we misschien 'yes' zouden moeten uitsluiten omdat dat betekent dat er copyright op zit en je het dus niet zomaar mag gebruiken.